test/e2e/settings/forwarded_headers.go

```diff
@@ -86,7 +86,7 @@ var _ = framework.DescribeSetting("use-forwarded-headers", func() {
 			Body().
 			Raw()
 
-		assert.Contains(ginkgo.GinkgoT(), body, "host=myhost.com")
+		assert.Regexp(ginkgo.GinkgoT(), `(\s)host=myhost`, body)
```

Comment:
Can you explain this?

Reply:
yes, because we check the entire response body, where each header is on a separate line, I had to rewrite a number of checks to make sure we are not matching on another header, namely the `x-original-forwarded-host=myhost` header, which itself contains the string `host=myhost`.

Reading the change again though I just noticed that I inadvertendly removed the tld (`.com`) of the check, I'll fix that.

Reply:
Ah, yes, sure, that makes sense! I didn't think of that! Can't you check for `^host=myhost` then and maybe add a comment that helps understand that little "challenge"? 🙂